packages/devtools_app/lib/src/performance/performance_controller.dart

```diff
@@ -418,7 +418,8 @@ class PerformanceController
     }
 
     if (offlinePerformanceData.cpuProfileData != null) {
-      cpuProfilerController.loadData(offlinePerformanceData.cpuProfileData);
+      cpuProfilerController
+          .loadProcessedData(offlinePerformanceData.cpuProfileData);
```

Comment:
Nit: trailing comma will format this more nicely... ;-)

Reply:
done 😆